### ISSUE TITLE
Add Make Offer option to NFT options dropdown

### ISSUE THREAD

**Issue by andrewhstovell** [xch-dev/sage#151]
Expected UX coming from reference GUI Wallet.
Current UX: User selects Offers -> Create Offer -> +NFT -> Realizes they need launcher id -> ... -> Offer is created. Quite tedious

**Comment by Rigidity** [comment]
Good idea, this is a related issue which will make the Offering assets a dropdown instead of a manual entry:
https://github.com/xch-dev/sage/issues/118

**Comment by Rigidity** [comment]
Multiselect makes this feature really useful

**Comment by Rigidity** [comment]
https://github.com/xch-dev/sage/pull/242
